tests/panels/test_sql.py

```diff
@@ -729,6 +729,16 @@ def test_similar_and_duplicate_grouping(self):
         self.assertNotEqual(queries[0]["similar_color"], queries[3]["similar_color"])
         self.assertNotEqual(queries[0]["duplicate_color"], queries[3]["similar_color"])
 
+    @unittest.skipUnless(
+        connection.vendor == "postgresql", "Test valid only on PostgreSQL"
```

Comment:
How come this is only valid for postgres? I feel like the other db drivers should also pass this test.

Reply:
From a quick look at the Django code, this can only happen if the DB backend has `supports_slicing_ordering_in_compound = True`. But I see now it is more widely supported than I thought. Shall I rather use
```python
@unittest.skipUnless(connection.vendor != "sqlite", "...")
```
?

Reply:
I'm not sure why the `skipUnless` is even necessary? The test should pass on all database engines anyway, or not?